internal/services/nginx/nginx_deployment_resource.go

```diff
@@ -178,8 +188,37 @@ func (m DeploymentResource) Arguments() map[string]*pluginsdk.Schema {
 				},
 			},
 		},
+		"scaling": {
+			Type:     pluginsdk.TypeList,
+			Required: true,
```

Comment:
I thought this was not a required field on the API? 🤔 At least on our API.

Reply:
It is not. I can make it optional.